lib/devise/multi_email/parent_model_extensions.rb

```diff
@@ -9,8 +9,12 @@ module ParentModelExtensions
 
       included do
         multi_email_association.include_module(EmailModelExtensions)
+
+        # Enable "autosave" to ensure email record changes are persisted
+        multi_email_association.reflection.options.merge!(autosave: true)
```

Comment:
@allenwq I will remove the `:autosave` option and make a note in README to add it if there is an issue. Would still be helpful to test and let me know if it works for you without that option explicitly set. Thanks.